src/sentry/snuba/subscriptions.py

```diff
@@ -143,24 +143,28 @@ def update_snuba_subscription(subscription, old_dataset):
     return subscription
 
 
-def bulk_delete_snuba_subscriptions(subscriptions):
+def bulk_delete_snuba_subscriptions(subscriptions, disable_subscription=False):
```

Comment:
Should you have a bulk_disable_snuba_subscriptions()?

Reply:
Yeah I can make this. I think that sounds more sane.